src/scenes/ProgressReports/EditForm/Steps/TeamNews/TeamNewsStep.tsx

```diff
@@ -40,15 +40,21 @@ export const TeamNewsStep: StepComponent = ({ report }) => {
 };
 TeamNewsStep.enableWhen = ({ report }) => report.teamNews.canRead;
 
-TeamNewsStep.isIncomplete = ({ report, currentUserRoles }) =>
-  report.teamNews.items[0]?.responses.some(
-    ({ variant: { responsibleRole }, response }) =>
-      // responsible
-      (responsibleRole ? currentUserRoles.has(responsibleRole) : false) &&
-      // empty and editable
-      !response.value &&
-      response.canEdit
-  ) ?? true;
+TeamNewsStep.isIncomplete = ({ report, currentUserRoles }) => {
+  const isIncomplete =
+    report.teamNews.items[0]?.responses.some(
+      ({ variant: { responsibleRole }, response }) =>
+        // responsible
+        (responsibleRole ? currentUserRoles.has(responsibleRole) : false) &&
+        // empty and editable
+        !response.value &&
+        response.canEdit
+    ) ?? true;
+  return {
+    isIncomplete: isIncomplete,
+    severity: NotificationSeverity.Suggested,
+  };
+};
```

Comment:
I think this looks cleaner
```suggestion
TeamNewsStep.isIncomplete = ({ report, currentUserRoles }) => ({
  isIncomplete:
    report.teamNews.items[0]?.responses.some(
      ({ variant: { responsibleRole }, response }) =>
        // responsible
        (responsibleRole ? currentUserRoles.has(responsibleRole) : false) &&
        // empty and editable
        !response.value &&
        response.canEdit
    ) ?? true,
  severity: 'suggested',
});
```